### ISSUE TITLE
Problem when instance solution is the empty subset.

### ISSUE THREAD

**Issue by smatchcube** [fontanf/teaching#1]
In the _batch scheduling_ project, when we try to check a certificate with an empty set of items we get an error because we are computing the `max` over an empty set. This happens for example in the first instance where we have no items (and solution with no items).
It's not clear from the subject what the maximum width means when no items are selected. Is it zero?
In this case adding the argument `default=0` to the the call of the `max` function should fix this:
```python
width = max((self.items[item_id].width
             for item_id in data["items"]), default=0)
```

**Comment by fontanf** [comment]
Yes, that's it. Thanks for reporting and sorry for the inconvenience. I think I need to fix this for the other subjects too

**Comment by fontanf** [comment]
Fixed in 9c70a4f6b9fe3ec4d359b1002af62e6402c2c57d
